spec/models/user_challenge_spec.rb

```diff
@@ -5,7 +5,7 @@
   let(:user_challenge) { create(:user_challenge) }
 
   describe "validations" do
-    it { is_expected.to validate_length_of(:notes).is_at_most(1500) }
```

Comment:
So you're testing note validations in the user_challenge spec rather than in note? Just curious why that's better, wouldn't that mean all user challenges have to have notes, or am I misunderstanding?

Reply:
Please ignore the 'user_challenge_spec' file and have a look at 'note_spec' file!